segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -358,11 +359,19 @@ private void handleException(UUID writerId, long requestId, String segment, long
             log.info("Closing connection '{}' while performing append on Segment '{}' due to {}.", connection, segment, u.getMessage());
             connection.close();
         } else {
-            log.error("Error (Segment = '{}', Operation = 'append')", segment, u);
+            logError(segment, u);
             connection.close(); // Closing connection should reinitialize things, and hopefully fix the problem
         }
     }
 
+    private void logError(String segment, Throwable u) {
+        if (u instanceof IllegalContainerStateException) {
+            log.warn("Error (Segment = '{}', Operation = 'append'): {}.", segment, u.toString());
```

Comment:
Is the `toString()` here reducing considerably the stacktrace?

Reply:
Here is a reference to the documentation:

https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#toString--

Reply:
Yes. `toString` will return the name of the class and the message. It will not include the stack trace.